src/Struct/FilterStruct.php

```diff
@@ -19,12 +21,32 @@ class FilterStruct {
     public $value;
 
     public function __construct($fieldName, $operator, $value) {
-        $this->fieldName = $fieldName;
+        if (strpos($fieldName, self::$LOGICAL_OR)) {
+            $this->fieldName = explode(self::$LOGICAL_OR, $fieldName);
+        } else {
+            $this->fieldName = $fieldName;
+        }
         $this->operator = $this->transformOperator($operator);
         $this->value = $value;
     }
 
     public function apply($object) {
+        if (is_array($this->fieldName)) {
+            $me = $this;
```

Comment:
@PMessinezis, don't you think this `$me` can deleted ?
It's not javascript anyway, so the callback won't go into another context like js.